### ISSUE TITLE
massive amounts of errors

### ISSUE THREAD

**Issue by Nordrassil** [Alogsdon/ZodsRaidAssign#8]
Message: Interface\AddOns\ZodsRaidAssign\ZodsRaidAssign.lua:37: attempt to index global 'ZRA_vars' (a nil value)
Time: Tue Mar 31 10:16:14 2020
Count: 1
Stack: Interface\AddOns\ZodsRaidAssign\ZodsRaidAssign.lua:37: attempt to index global 'ZRA_vars' (a nil value)
Interface\AddOns\ZodsRaidAssign\ZodsRaidAssign.lua:37: in function `mapRaids'
Interface\AddOns\ZodsRaidAssign\ZodsRaidAssign.lua:65: in function `onLoad'
Interface\AddOns\ZodsRaidAssign\ZodsRaidAssign.lua:168: in function <Interface\AddOns\ZodsRaidAssign\ZodsRaidAssign.lua:165>

Locals: m = <table> {
 Roles = "0"
}

Message: Interface\AddOns\ZodsRaidAssign\ZodsRaidAssign.lua:189: attempt to index global 'ZRA_vars' (a nil value)
Time: Tue Mar 31 10:16:18 2020
Count: 1
Stack: Interface\AddOns\ZodsRaidAssign\ZodsRaidAssign.lua:189: attempt to index global 'ZRA_vars' (a nil value)
[C]: ?
Interface\AddOns\ZodsRaidAssign\ZodsRaidAssign.lua:189: in function <Interface\AddOns\ZodsRaidAssign\ZodsRaidAssign.lua:165>

Locals: 

Message: Interface\AddOns\ZodsRaidAssign\ZRA_UI.lua:8: attempt to index global 'ZDragframe' (a nil value)
Time: Tue Mar 31 10:17:03 2020
Count: 2331
Stack: Interface\AddOns\ZodsRaidAssign\ZRA_UI.lua:8: attempt to index global 'ZDragframe' (a nil value)
[C]: ?
Interface\AddOns\ZodsRaidAssign\ZRA_UI.lua:8: in function <Interface\AddOns\ZodsRaidAssign\ZRA_UI.lua:7>

Locals: 

Message: ...terface\AddOns\AddOnSkins\Skins\AddOns\Outfitter.lua:7: attempt to index global 'PaperDollSidebarTabs' (a nil value)
Time: Tue Mar 31 10:16:30 2020
Count: 1
Stack: ...terface\AddOns\AddOnSkins\Skins\AddOns\Outfitter.lua:7: attempt to index global 'PaperDollSidebarTabs' (a nil value)
[C]: ?
...terface\AddOns\AddOnSkins\Skins\AddOns\Outfitter.lua:7: in function <...terface\AddOns\AddOnSkins\Skins\AddOns\Outfitter.lua:7>
[C]: ?
[C]: in function `Show'
Interface\FrameXML\UIParent.lua:2087: in function `SetUIPanel'
Interface\FrameXML\UIParent.lua:1932: in function `ShowUIPanel'
Interface\FrameXML\UIParent.lua:1793: in function <Interface\FrameXML\UIParent.lua:1789>
[C]: in function `SetAttribute'
Interface\FrameXML\UIParent.lua:2535: in function `ShowUIPanel'
Interface\FrameXML\CharacterFrame.lua:26: in function `ToggleCharacter'
[string "TOGGLECHARACTER0"]:1: in function <[string "TOGGLECHARACTER0"]:1>

Locals:

**Comment by Alogsdon** [comment]
yeah. I'm aware this can happen. I'm working on a large overhaul right now. in the mean time "/zra clear" and a reload should make the errors go away

**Comment by speaker** [comment]
Cannot run "/zra clear" because the addon is stuck in an infinite loop and will not accept events to trigger processing.
